app/components/plans/newplan.js

```diff
@@ -1,143 +1,206 @@
-   "vid_url" : ""
-  },
-  { "id" : 3,
-    "title" : "Prone External Rotation",
-    "description" : "Start by lying on your stomach and your arm dangling",
-    "img_URL" : "",
-    "vid_url" : ""
-  }
-];
-//============================
+//Stateless Components
+import PlanOptions from './newPlanOptions';
+import PatientPanel from './PatientPanel';
+import Treatment from './treatment';
+import CreatedTreatments from './createdTreatments';
+
+//material-ui
+import {StepsRaisedButton, StepsFlatButton} from '../material-style';
+import {MenuItem, Divider, FloatingActionButton, TextField, SelectField, Link} from 'material-ui';
+
+// Styles
+//== Exerise SelectField Style
+const styleRow = {
+  'display' : 'flex',
+  'marginTop' : '1em'
+};
+// Initial Treatment Templete for reseting
+const initialTreatment = {
+  time_per_exercise: '',
+  reps: '',
+  sets: '',
+  resistance: '',
+  notes: ''
+};
+
 //=======  Component==========
-class newPlan extends React.Component{
+export default class newPlan extends React.Component{
   constructor(props){
     console.log(props);
     super(props);
 
     this.state={
-      patient : {},
-      duration : 1,
-      injury : null,
-      notes : null,
-      exercise: "",
-      exercises : []
+      duration : 0,
+      therapy_focus : "",
+      notes : "",
+      selectedExercise : {},
+      treatment: initialTreatment,
+      treatments : []
     };
-    this.handleChange = this.handleChange.bind(this);
+
+    this.durationOnChange = this.durationOnChange.bind(this);
+    this.therapyHandler = this.therapyHandler.bind(this);
+    this.noteHandler = this.noteHandler.bind(this);
+    this.exerciseOnChange = this.exerciseOnChange.bind(this);
+    this.notesOnChange = this.notesOnChange.bind(this);
+    this.treatmentHandler = this.treatmentHandler.bind(this);
     this.submitHandler = this.submitHandler.bind(this);
+    this.addNewTreatment = this.addNewTreatment.bind(this);
+    this.removeTreatment = this.removeTreatment.bind(this);
+  }
+//==== Plan Option Handlers
+// persisting on local state for plan for duration and injury
+  durationOnChange(evt, idx, value) {
+    this.setState({ duration: value });
   }
 
-  // persisting on local state
-  handleChange(evt,index, value){
+//onchange handler for therapy_focus
+  therapyHandler(evt) {
+    this.setState({therapy_focus : evt.target.value});
+  }
 
+//onchange handler for PlanNotes
+  noteHandler(evt) {
+    this.setState({notes : evt.target.value});
+  }
+// Treatment Handlers
+// handle change to this.state.exercise
+  exerciseOnChange(evt, idx) {
+    this.setState({ selectedExercise: this.props.exercises[idx] });
   }
 
-  //submiting local state
-  submitHandler(evt){
+  notesOnChange(evt) {
+    this.setState({ treatment: { notes: evt.target.value }});
+  }
 
+// treatment handler for this.state.treatment
+  treatmentHandler(field, value) {
+    this.setState({ treatment :{[field]: value }});
```

Comment:
Aha! So this syntax works? This was the bug I didn't get to before pushing it back to you.